versioned_docs/version-v6.0.0/framework/pubsubV2.md

```diff
@@ -37,27 +50,60 @@ If the request content length exceeds a certain size (default is `20 0000 bytes`
   }
 }
 ```
+</details>
+
+The endpoint will respond with `202 Accepted`, if the message got published successfully.
 
-## Subscribe on multiple messages
-The subscribe function is used to receive messages from the specified Service Bus subscription. If the subscription does not yet exist, it will be automatically created with the specified SQL filter rule. If a message contains the claim check properties then the content is retrieved from Azure Storage.
+## ⬅️ Subscribe for messages
+The `/api/Subscribe` endpoint allows users to periodically ask for any available published messages on the configured Azure Service Bus topic (default: `pubsubv2router`).
 
-Once a message has been received, it is then deferred so that the deferred message can be picked up for use within the Acknowledge function.*
+The following request properties are available during subscription:
```

Comment:
Might be better to rename to `The following request properties can be supplied`?

Currently it seems a bit like these properties are available but we actually need to supply them ourselves

Reply:
Made it `must/can be supplied` as otherwise, the `can` is also invalid in case of required properties.
Changes made here for all: https://github.com/invictus-integration/docs-ifa/pull/378/commits/776b21304079138bd7984f1fc3c9c0049e69f354